include/deal.II/matrix_free/tools.h

```diff
@@ -443,7 +443,188 @@ namespace MatrixFreeTools
                      }),
               locally_relevant_constrains.end());
 
-            // STEP 2c: translate COO to CRS
+            // STEP 2b: apply hanging-node constraints
```

Comment:
As we already did step 2b above, would this be
```suggestion
            // STEP 2c: apply hanging-node constraints
```